include/openssl/pkcs7.h

```diff
@@ -498,6 +498,11 @@ PKCS7_get_recipient_info(PKCS7 *p7);
 OPENSSL_EXPORT OPENSSL_DEPRECATED PKCS7_RECIP_INFO *PKCS7_add_recipient(
     PKCS7 *p7, X509 *x509);
 
+// PKCS7_get0_signers retrieves the signer's certificates from p7. It does not
+// check their validity or whether any signatures are valid
```

Comment:
we should note here that the caller takes ownership of the `STACK_OF(X509)*` and is responsible for freeing it.